src/utils/document-style-utils.ts

```diff
@@ -0,0 +1,10 @@
+import { StoredStyle } from './table-of-contents-utils';
+
+export function getStyleLevel(styleName: string, styles: StoredStyle[]): number | null {
+  const style = styles.find(style => style.name === styleName);
+
+  if (!style?.level) return null;

```

Comment:
should prefer undefined to null. Null should only be used if it's meaning is distinct from undefined.

Reply:
Null is correct in this case as the types where the value is used expect null or a number and the attribute is not optional..